src/Texts/gameplayTexts.js

```diff
@@ -18,6 +18,7 @@ const gameplayTexts = {
   },
   actionsButtonTexts: {
     title: 'Actions',
+    moreActions : 'More Actions'
   },
```

Comment:
this section relates to the texts of the actionsButton component.
please create a new key called actionsTabTexts.